lib/raygun.ts

```diff
@@ -185,10 +186,45 @@ class Raygun {
     return raygunTransport;
   }
 
-  send(
+  /**
+   * Sends exception to Raygun.
+   * @param exception to send.
+   * @param customData to attach to the error report.
+   * @param request custom RequestParams.
+   * @param tags to attach to the error report.
+   * @return IncomingMessage if message was delivered, null if stored, rejected with Error if failed.
+   */
+  async send(
     exception: Error | string,
     customData?: CustomData,
-    callback?: (err: Error | null) => void,
+    request?: RequestParams,
+    tags?: Tag[],
+  ): Promise<IncomingMessage | null> {
+    // Convert internal sendWithCallback implementation to a Promise.
+    return new Promise((resolve, reject) => {
+      this.sendWithCallback(
+        exception,
+        customData,
+        function (err, message) {
+          if (err != null) {
+            reject(err);
+          } else {
+            resolve(message);
+          }
+        },
+        request,
+        tags,
+      );
+    });
+  }
+
+  /**
+   * @deprecated use send instead, which supports async/await calls.
```

Comment:
```suggestion
   * @deprecated use sendWithCallback instead, which supports async/await calls.
```

Is this suggestion correct?